app/src/main/res/layout/activity_main.xml

```diff
@@ -55,6 +55,16 @@
             android:text="@string/start"
             android:textAlignment="center"
             android:textSize="20sp" />
+        <ListView
```

Comment:
this comment is for 3 aboves image resources androiicon.png, iosicon.jpg option.png 

This is dangerous, images in `drawable-v24` only use for device API 24, other API will be crash or distored!
Put these resource in drawable-xhdpi